### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by Yuejiang-li** [desktop/desktop#11111]
Hi, I have used Github Desktop fine for a long time. But yesterday, I encountered the following issue:
```
RangeError: Maximum call stack size exceeded
    at Rm (file:///app/renderer.js:59:16576)
    at file:///app/renderer.js:59:15957
    at file:///app/renderer.js:45:243596
    at Em.render (file:///app/renderer.js:59:11792)
    at ko (file:///app/renderer.js:37:62856)
    at Eo (file:///app/renderer.js:37:62651)
    at Do (file:///app/renderer.js:37:66489)
    at Bs (file:///app/renderer.js:37:90287)
    at Hs (file:///app/renderer.js:37:90671)
    at ka (file:///app/renderer.js:37:97697)

Version: 2.6.0
OS: Windows 10.0.18363
```
The weird thing is that I cannot close this window, only if I directly close my computer.

I have tried to uninstall Github Desktop, and reinstall it. However, this issue still exists. It suggests that *GitHub Desktop has encountered an unrecoverable error and will need to restart. This has been reported to the team, but if you encountered this repeatedly please report this issue to the Github Destop issue tracker*. So I came here to look for some help.

**Comment by niik** [comment]
Hi @Yuejiang-li, sorry to hear you're having problems. The crash you're experiencing is coming from the new split diff mode that we released in 2.6.0. It's been fixed in #11064 but hasn't shipped to production yet.

I'm not sure why exactly you're seeing this problem but in our testing we found that complex (i.e most often large) diffs are the culprit. In order to work around the problem you can either attempt to manually commit or stash your changes in your repository before opening GitHub Desktop (and then switching back to the unified diff mode) or you can try resetting GitHub Desktop by following these steps

- Close GitHub Desktop
- Remove all of the files in %LocalAppData%\GitHub Desktop\
- Open GitHub Desktop  again

Let us know if that doesn't get things working. Please note that resetting means that you will have to sign in again and add your repositories.

**Comment by Yuejiang-li** [comment]
Yes, I indeed have some big changes which have not staged. I remember that I happened to touch the diff mode button that day. This issue had been addressed. Thank you so much!
